src/resources/postcss/base/full/forms/_checkboxes-radios.pcss

```diff
@@ -66,13 +66,16 @@
 	.tribe-common-form-control-checkbox,
 	.tribe-common-form-control-radio {
 		line-height: 0;
+
+		&:focus,
+		&:hover {
+			opacity: .8;
```

Comment:
We have a variable for opacity, let's use that. Also, `.tribe-common-form-control-radio` and `.tribe-common-form-control-checkbox` are divs, non-focusable by design. The `:hover` and `:focus` should be on a hoverable/focusable element, which would be the input.